RasimKhusaenov/my_own_blog
# Add Docker

### Summary

- Added Dockerfile
- Added `docker-compose.yml`
- Added scripts: `bin/wait-for`, `bin/docker-entrypoint`, `bin/docker-server`
- Added `.env`
- Added `.dockerignore`
- Updated `.gitignore`

### How it works

Screenshots/screencasts of the pull request introduced functionality.

### Test plan

List of steps to manually test introduced functionality:

- Run `bin/docker-server`
- Go to http://localhost:5000
- Run `docker-compose exec web rspec`
- Make sure that all works fine

### Deploy notes

- Create `.env` file
- Add `DATABASE_HOST`, `DATABASE_USERNAME`, `DATABASE_PASSWORD`, `REDIS_URL`

### Checklist:


### References
N/A
